tests/test_contacts_book.py

```diff
@@ -31,3 +31,14 @@ def test_contacts_book_returns_all_contacts(self):
         contacts = contacts_book.get_contacts()
 
         assert len(contacts) == 2
+
+    def test_contacts_book_removes_contacts(self):
```

Comment:
It's smelly to have tests that assert on something, do something else and do another assert.
Those can often be separate and more focussed tests.

In this instance it looks like there's one test that should verify that the length of the address book after adding an entry is one.

And the other test would be that adding an entry and removing it again, leaves the address book empty (i.e. `len` is `0`)